xcache/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM opensciencegrid/software-base:fresh
 LABEL maintainer OSG Software <help@opensciencegrid.org>
 
 # Default root dir
-ENV ROOTDIR /mnt/stash
+ENV XC_ROOTDIR /xcache/namespace
```

Comment:
Don't you just want to remove this?
If not, you should probably use it in the `mkdir -p` line below.